server/views/pages/component.njk

```diff
@@ -23,6 +23,71 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 data-test="detail-page-title">{{ component.name }}</h1>
+    {% if component.alerts | length %}
+      <div class="govuk-inset-text govuk-!-margin-bottom-4" data-test="alerts" style="border-left: 5px solid #d4351c; background-color: #fef1f0;">
+        <h2 class="govuk-heading-m">Active alerts</h2>
+        <table class="govuk-table" style="table-layout: fixed; width: 100%;">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th class="govuk-table__header">Alert Name</th>
+              <th class="govuk-table__header">Environment</th>
+              <th class="govuk-table__header">Summary</th>
+              <th class="govuk-table__header">Message</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            {% set firstAlert = component.alerts[0] %}
```

Comment:
Still confused about the logic around different displays if there are 1 or 2+ alerts?